spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/trace/DefaultTracer.java

```diff
@@ -242,7 +228,7 @@ private Span createContinuedSpan(Span span, Span saved) {
 		if (saved == null && span.getSavedSpan() != null) {
 			saved = span.getSavedSpan();
 		}
-		return new Span(span, saved);
```

Comment:
so looking at the code and toying around, I think the best is to continue use of the old constructor here and not add a new public method of buildContinuedSpan. In sleuth 2, we will almost surely change this part of the code, so it is helpful to avoid having to deprecate another thing between now and then